src/microcosm.js

```diff
@@ -177,7 +180,14 @@ export default class Microcosm extends Emitter {
       config = { register: config }
     }
 
-    this.stores = this.stores.concat([[ key, config ]])
+    this.domains = this.domains.concat([[ key, config ]])
+
+    for (const domain of this.domains) {
+      const [key, config] = domain
+      if (key) {
```

Comment:
duplicate/overwrites were allowed before with the metaStore (by calling `addStore` with no `key` argument, once could replace the default metaStore). Is that something we want to continue to allow?